### PR TITLE
[4.0] Simplify code

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

Removes some unnecessary ternary operators.

### Testing Instructions

Code review.

### Documentation Changes Required

No.

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 2dee50cb69dce8b0183d387e9b1cf29db9b1a003<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26339">issues.joomla.org/tracker/joomla-cms/26339</a>.</sub>

File: administrator/components/com_media/tmpl/media/default.php [hunk 1]
```diff
@@ -50,7 +50,7 @@
 	'maxUploadSizeMb'         => $params->get('upload_maxsize', 10),
 	'providers'               => (array) $this->providers,
 	'currentPath'             => $this->currentPath,
-	'isModal'                 => Factory::getApplication()->input->getCmd('tmpl', '') === 'component' ? true : false,
+	'isModal'                 => $tmpl === 'component',
```
Comment (Quy) [inline administrator/components/com_media/tmpl/media/default.php:53]: Apply strict comparison on line 36?

Comment (SharkyKZ) [reply administrator/components/com_media/tmpl/media/default.php:53]: Done.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 6ece35d6206aeab32ad7730a13844623d37d94d0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26339">issues.joomla.org/tracker/joomla-cms/26339</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26339">issues.joomla.org/tracker/joomla-cms/26339</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**